src/classes/PS_CommitmentRequest.cls

```diff
@@ -178,16 +214,22 @@ public inherited sharing class PS_CommitmentRequest {
     */
     @TestVisible
     private void setDonorInfo(RequestBody reqBody, npe03__Recurring_Donation__c rd) {
-        Boolean isContactDonor = rd.npe03__Contact__c != null
+        Contact contactDonor = rd.npe03__Contact__c != null
```

Comment:
Need to query Contact and Account for the donor names since those values are not available on the RD record constructed from the entry form fields. (W-8554656)